cli/cli_app.go

```diff
@@ -451,15 +452,23 @@ func runTerragrunt(terragruntOptions *options.TerragruntOptions) (finalStatus er
 		}
 	}
 
-	if err := conf.ImportFiles.Run(err, foldersWithTerraformFiles...); stopOnError(err) {
-		return
+	if folders := foldersWithTerraformFiles[1:]; len(folders) > 0 {
```

Comment:
This is the only part (with the substitutions below) that's not described in the PR description but it's the most interesting. What does it do?

Reply:
We already import files in the current folder. This code avoid importing the files twice and only process the sub-folders.

Reply:
Definitely needs a comment

Reply:
Regarding the substitutions. Sorry, I splitted my (as usual) too big PR into smaller PRs and I forgot to isolate that part. The goal is to remove terraform warnings when we generate code that is not 100% terraform 0.12 compliant such as:
`provider = "aws.@default_region"
that will become:
`provider = "aws.us-east-1"`
and will be transformed by this substitution to:
`provider = aws.us-east-1`